### ISSUE TITLE
Add padding to checkbox and radio in horizontal form (v4)

### ISSUE THREAD

**Issue by Holt59** [twbs/bootstrap#17211]
When using radio and checkbox in `horizontal` form, the currently specified format is the following (from http://v4-alpha.getbootstrap.com/components/forms/#using-the-grid):
```html
<form>
  <div class="form-group row">
    <label for="inputEmail3" class="col-sm-2 form-control-label">Email</label>
    <div class="col-sm-10">
      <input type="email" class="form-control" id="inputEmail3" placeholder="Email">
    </div>
  </div>
  <div class="form-group row">
    <label class="col-sm-2">Radios</label>
    <div class="col-sm-10">
      <div class="radio">
        <label>
          <input type="radio" name="gridRadios" id="gridRadios1" value="option1" checked>
          Option one is this and that&mdash;be sure to include why it's great
        </label>
      </div>
      <div class="radio">
        <label>
          <input type="radio" name="gridRadios" id="gridRadios2" value="option2">
          Option two can be something else and selecting it will deselect option one
        </label>
      </div>
    </div>
  </div>
  <div class="form-group row">
    <label class="col-sm-2">Checkbox</label>
    <div class="col-sm-10">
      <div class="checkbox">
        <label>
          <input type="checkbox"> Check me out
        </label>
      </div>
    </div>
  </div>
</form>
``` 
Checkbox and radio label does not have a `form-control-label` class, so they have a little left offset ( `padding-left: 0.9375rem` instead of `padding-left: 0.75rem`) and the spacing between all the labels is not equal (`margin-bottom: 0.5rem` for radio/checkbox label and `padding: 0.4375rem` for the others).

I think that `form-control-label` should be required even for checkbox and radio group, but currently, adding this class create an offset between the label and the radio/checkbox, so I propose to add proper `padding-top` and `padding-bottom` property to `radio` and `checkbox` div in horizontal form:
```css
.form-horizontal .checkbox,
.form-horizontal .radio {
    padding: 0.4375rem 0;
    margin-bottom: 0;
}
```

**Comment by zacechola** [comment]
Maybe I'm misinterpreting you, but I'm not sure that'd work. Adding top padding to the first radio/checkbox offsets the baseline of the radios/checkboxes from the label baselines in the example.

[Example](http://jsbin.com/kotati/edit?html,css,output)

**Comment by Holt59** [comment]
What I proposed was to add `form-control-label` to the `label` tag used for `checkbox` and `radio` in order to left-align them with the other labels and to have a regular spacing between labels (currently, `checkbox` and `radio` label have a little left offset and the spacing is not regular because standard `label` have `padding-top` and `padding-bottom` while `label` for `checkbox` and `radio` have `margin-bottom`).

**But** if you add the `form-control-label` class, the `label` is not aligned with the radio/checkbox, so a `padding` should be added to the `.checkbox` and `.radio` div.

[Example](http://jsbin.com/kuhuha/1/edit?html,css,output)

I simply added `form-control-label` to the `label` tags for checkbox/radio and `form-horizontal` to the `form` tag.
